optaweb-vehicle-routing-frontend/package.json

```diff
@@ -30,7 +30,9 @@
     "eject": "react-scripts eject",
     "coverage": "npm run test -- --coverage --watchAll=false",
     "lint": "./node_modules/eslint/bin/eslint.js ./src --ext .ts,.tsx",
-    "lint:fix": "npm run lint -- --fix"
+    "lint:fix": "npm run lint -- --fix",
+    "cypress:open": "cypress open",
+    "cypress:run": "cypress run"
```

Comment:
What do you think about shortening this to `cy:run`, so I'd only need to type `npm run cy:run` on command line?

Reply:
I just found out this is suggested in the [Cypress documentation](https://docs.cypress.io/guides/getting-started/installing-cypress.html#Adding-npm-scripts), so let's not diverge from it.